R/derive_locf_records.R

```diff
@@ -109,7 +110,9 @@
 #'   by_vars = exprs(STUDYID, USUBJID, PARAMCD),
 #'   order = exprs(AVISITN, AVISIT),
 #'   keep_vars = exprs(PARAMN)
-#' )
+#' ) |>
+#'   arrange(desc(USUBJID), PARAMCD, AVISIT) |>
```

Comment:
Rather than this, I would modify the original ADVS (lines 70-88) to switch around the USUBJIDs so that you don't then have to sort by descending USUBJID. Also then `print(n=30)` may not be required.